src/tests/edje/data/meson.build

```diff
@@ -25,7 +25,8 @@ foreach edc_file : edc_files
    themes += custom_target('edje_cc_' + edc_file,
     input : edc_file,
     output : '@BASENAME@.edj',
-    command : edje_cc_exe + [ '-beta', '-fastcomp',
+    command : edje_cc_exe + [ '-beta', '-fastcomp', '-in-tree',
+              '-id', join_paths(meson.source_root(), 'src', 'tests', 'data'),
```

Comment:
@JPTIZ here it is another odd change

Reply:
this probably should be to remove emotion too

Reply:
> actually there is no src/tests/data in tree, I'll keep only src/tests/emotion/data here and everywhere else.